### ISSUE TITLE
extension doesn't support multiple kinds in a single file

### ISSUE THREAD

**Issue by VickyWinner** [redhat-developer/vscode-tekton#580]
I am using a single `.yaml` file to create all the resources required for the pipeline by separating them with `---`.  I get error after I installed this extension.  Not sure if its by design or a bug.

```
yaml: expected a single document in the stream, but found more
```

**Comment by sudhirverma** [comment]
> ```
> yaml: expected a single document in the stream, but found more
> ```

Hi, @VickyWinner this error you get while creating multiple resources in a single file? If yes which command you were running to create resources.

**Comment by VickyWinner** [comment]
@sudhirverma just take any resource required by the pipeline.  Task, TaskRun, Pipeline, PipelineRun etc.

**Comment by evidolob** [comment]
@VickyWinner I cannot to reproduce this, could you share more detailed steps to reproduce?

**Comment by VickyWinner** [comment]
its not erroring out while running the `kubectl` or `oc`.  Its just the vscode pops errors.

**Comment by evidolob** [comment]
I try this https://github.com/tektoncd/pipeline/blob/main/examples/v1beta1/pipelineruns/pipelinerun.yaml and havent see any errors

**Comment by VickyWinner** [comment]
this is what I see
![image](https://user-images.githubusercontent.com/33285331/120647436-9b78b280-c448-11eb-92f8-8ed883960419.png)

**Comment by evidolob** [comment]
@VickyWinner Are you using latest version of vscode-tekton?

**Comment by VickyWinner** [comment]
I am using `v0.14.0`

**Comment by evidolob** [comment]
Latest is `0.16.1` try that version

**Comment by VickyWinner** [comment]
I just did install `0.16.1` but still the same issue.  just an update that I am running vscode in administrator mode.

**Comment by VickyWinner** [comment]
No.  I am using only `.yaml` extension for all tekton stuff.  I ran vscode in non-administrator mode, same issue. looks like it has nothing to do with that.

**Comment by evidolob** [comment]
I mean any other vscode extensions(plugins) which can handle/provide some editor features for yaml files.
